### PR TITLE
Fix errors being hidden in templates

### PR DESCRIPTION
**Pull request by yaakovLowenstein**

### CODE REVIEW

**Comment by yaakovLowenstein** [comment]
Solved https://github.com/clokep/django-render-block/issues/51

File: tests/tests.py [hunk 1]
```diff
@@ -91,6 +91,11 @@ def test_subblock_no_parent(self):
         result = render_block_to_string("test_sub.html", "first")
         self.assertEqual(result, "\nbar\n")
 
+    @override_settings(DEBUG=True)
+    def test_exceptions_in_templates(self):
+        with self.assertRaises(Exception):
```
Comment (clokep) [inline tests/tests.py:96]: Should we use a custom execution class to ensure it is the expected exception that we're throwing?

**Review by clokep — APPROVED** [review]
This fixes the unit tests and seems to better match what Django internally does.

**Comment by yaakovLowenstein** [comment]
Thanks for merging!

**Pull request merged**